### PR TITLE
[DOCS] Upgrade docusaurus 3.0

### PR DESCRIPTION
**Pull request by JessSaavedra**
- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

**Review by josectobar — COMMENTED** [review]
This is looking good. Let me know once the issues with the build are addressed and if you need a hand with that.

File: docs/docusaurus/docs/core/installation_and_setup/manage_data_contexts.md [hunk 1]
```diff
@@ -181,12 +181,9 @@ If you're using GX for multiple projects, you might want to use a different Data
 
 ### Prerequisites
 
-<Prerequisites requirePython = {false} requireInstallation = {true} requireDataContext = {false} requireSourceData = {null} requireDatasource = {false} requireExpectationSuite = {false}>
-
+- A Great Expectations instance.
```
Comment (JessSaavedra) [inline docs/docusaurus/docs/core/installation_and_setup/manage_data_contexts.md]: I think I should add a link here but I was not sure how to write it correctly in markdown so I just deleted it (it used to be a `VersionedLink` in the prerequisites component). Please @Rachel-Reverie @kwcanuck feel free to tell me to update it as you think it should be

Comment (Rachel-Reverie) [inline docs/docusaurus/docs/core/installation_and_setup/manage_data_contexts.md]: For now you can use the following format:
```suggestion
- [A Great Expectations instance](/core/installation_and_setup/install_gx.md).
```
In the future this doc will be revised to be in line with the format we settled on for the "Manage Expectations" topic -- we'll update it to use the standardized, imported prerequisites then.

**Review by kwcanuck — APPROVED** [review]
Thanks for your effort on this!

**Review by josectobar — APPROVED** [review]
Great work

**Pull request merged**